packages/CodingStyle/src/Rector/Use_/RemoveUnusedAliasRector.php

```diff
@@ -299,17 +292,55 @@ private function resolveTraitUseNames(Node $searchNode): void
         }
     }
 
-    private function resolveDocPossibleAliases(Node $searchNode): void
+    /**
+     * @return string[]
+     */
+    private function resolveDocPossibleAliases(Node $searchNode): array
     {
-        $this->traverseNodesWithCallable($searchNode, function (Node $node): void {
+        $possibleDocAliases = [];
+
+        $this->traverseNodesWithCallable($searchNode, function (Node $node) use (&$possibleDocAliases): void {
             if ($node->getDocComment() === null) {
                 return;
             }
 
-            $matches = Strings::matchAll($node->getDocComment()->getText(), '#\@(?<possible_alias>.*?)\\\\#s');
+            $matches = Strings::matchAll($node->getDocComment()->getText(), '#\@(?<possible_alias>\w+)(\\\\)?#s');
```

Comment:
This line was the bugfix. Appearanbly, annotations without following `\\` broke this:

```php
* @UniqueEntity()
* @ThisWasSkipped
```

Reply:
Good catch! :)